doc/code/qml_resources.rst

```diff
@@ -0,0 +1,15 @@
+qml.resources
+=============
+
+Overview
+--------
+
+The resources module provides the functionality to estimate the cost of implementing quantum
```

Comment:
whatever word we pick, it should be the name of the module :) that way we have a singular name to refer to long-term/future/fault-tolerant algorithms in PennyLane that we can use _consistently_, that in the users-mind communicates "I can play and build with these now, but they aren't necessarily simulatable or hardware-executable _yet_".

Reply:
This way we frame it so that the user is _excited_ for the future, rather than being _disappointed_ that this doesn't integrate/work how they expect _now_.